src/org/loklak/api/search/LocationWiseTimeService.java

```diff
@@ -59,12 +58,16 @@ public JSONObject getDefaultPermissions(BaseUserRole baseUserRole) {
 		return null;
 	}
 
-	@Override
-	public JSONObject serviceImpl(Query call, HttpServletResponse response, Authorization rights,
-			JSONObjectWithDefault permissions) throws APIException {
-		String query = call.get("query", "");
-		return locationWiseTime(query);
-	}
+    @Override
+    public JSONObject serviceImpl(
+            Query call,
+            HttpServletResponse response,
+            Authorization rights,
+            JSONObjectWithDefault permissions
+    ) throws APIException {
+        String query = call.get("query", "");
+        return locationWiseTime(query);
+    }
```

Comment:
Maybe it's about the varying taste of code styling. But since it didn't violate any rules, I think it shouldn't be changed.